pkgs/development/python-modules/graph-tool/default.nix

```
@@ -64,13 +84,15 @@ buildPythonPackage rec {
     pygobject3
     scipy
     sparsehash
-  ];
+  ] ++ lib.optional stdenv.isDarwin llvmPackages.openmp;
```

Non blocking comment

```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
    llvmPackages.openmp
  ];
```